lightspeed/modules/proc_create-connection-secrets.adoc

```diff
@@ -0,0 +1,89 @@
+:_content-type: PROCEDURE
+
+[id="create-connection-secrets_{context}"]
+
+= Creating connection secrets
+
+Use this procedure to create secrets to connect to {PlatformName} and {ibmwatsonxcodeassistant}.
+
+.Prerequisites
+* You have installed the {PlatformNameShort} operator on the {OCP}. 
+* You have created an OAuth application in the automation controller.
```

Comment:
automation controller uses an attribute: {ControllerName}

Reply:
Done, thanks for pointing it out.